prometheus-libvirt-exporter.go

```diff
@@ -3,17 +3,21 @@ package main
 import (
 	"encoding/xml"
 	"flag"
+	"fmt"
+	"net"
+	"net/http"
+	"time"
+
 	"github.com/digitalocean/go-libvirt"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/zhangjianweibj/prometheus-libvirt-exporter/libvirt_schema"
-	"log"
-	"net"
-	"net/http"
-	"time"
+	"go.uber.org/zap"
```

Comment:
hi cavus700:
    there are many log tools ,zap is an extra tool compare with golang log.is it appropriate to introduce zap?

Reply:
I do not have a strong opinion on this but I thought it would be nice to have some log levels and zap is really easy to use. If you have any suggestions or do not like it I can change/remove it.